### PR TITLE
WRO-567: Added missing screenshot tests for focused state

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added missing screenshot tests for focused state

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-567

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: tests/screenshot/apps/components/ProgressButton.js [hunk 1]
```diff
@@ -93,7 +93,71 @@ const LtrTests = [
...
+		<ProgressButton backgroundOpacity="opaque" color="yellow" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="blue" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="red">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="green">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="yellow">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="blue">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="red" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="green" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="yellow" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="opaque" color="blue" minWidth={false}>Focused Update</ProgressButton>,
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/ProgressButton.js:141]: These tests already worked in line 130~133.

File: tests/screenshot/apps/components/ProgressButton.js [hunk 2]
```diff
@@ -93,7 +93,71 @@ const LtrTests = [
...
+		<ProgressButton backgroundOpacity="transparent" color="yellow" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="blue" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="red">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="green">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="yellow">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="blue">Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="red" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="green" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="yellow" minWidth={false}>Focused Update</ProgressButton>,
+		<ProgressButton backgroundOpacity="transparent" color="blue" minWidth={false}>Focused Update</ProgressButton>
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/ProgressButton.js:159]: These tests already worked in line 148~151.

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**